tests/node/test.js

```diff
@@ -2,17 +2,17 @@
 'use strict';
 
 const path = require('path');
-const expect = require('chai').expect;
+const assert = require('chai').assert;
 const jsdom = require('jsdom');
 const Module = require('module');
 const sinon = require('sinon');
 
 const _require = Module.prototype.require;
 
-describe('node acceptance tests', function () {
+suite('node acceptance tests', function () {
   let sandbox;
 
-  before(function () {
+  suiteSetup(function () {
```

Comment:
Should be. I used [this](https://github.com/mochajs/mocha/blob/2bb2b9fa35818db7a02e5068364b0c417436b1af/lib/interfaces/tdd.js#L40-L43) as reference.